src/postselection.jl

```diff
@@ -0,0 +1,266 @@
+"""
+    initByMsmt(data, numSegments, numMeas, indMeas, selectMeas, selectSign, threshold, docond)
+
+Condition data on measurement results on the same channel
+
+numSegments = number of distinct segments in the sequence
+numMeas = number of meauserements per segments
+indMeas = list of indeces of the msmt's to compare for fidelity
+selectMeas = indeces of msms't to postselect on. All the msm'ts in a
+segment will be postselected on this one
+selectSign = select the sign for postselection. 1 = keep greater than;
+-1 = keep smaller than
+threshold
+docond. If 1, do postselection
+"""
+function initByMsmt(data,numSegments,numMeas,indMeas,selectMeas,selectSign,threshold,docond)
+
+  ind0=indMeas[1,:]
+  ind1=indMeas[2,:]
+  numShots = Int64(length(data)/(numMeas*numSegments))
+  data=real(data)'
+  datamat = splitdata(data,numShots,numMeas*numSegments)
+  bins = linspace(minimum(minimum(datamat)), maximum(maximum(datamat)),500)
```

Comment:
In Julia, `minimum` and `maximum` iterate over the entire input, so the doubled `minimum` and `maximum` are superfluous. In fact, you can get both at once with `extrema`. So, I would re-write this as
```julia
bins = linspace(extrema(datamat)..., 500)
```